src/components/Content.jsx

```diff
@@ -0,0 +1,45 @@
+import { Route, Routes } from 'react-router-dom';
+import { ArticleList } from '../pages/ArticleList';
+import { MostRecentArticles } from '../pages/MostRecentArticles';
+import { useEffect, useState } from 'react';
+import { getAllArticles } from '../utils/api';
+
+export const Content = () => {
+  const [articleList, setArticleList] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getAllArticles()
+      .then(({ articles }) => {
+        setArticleList(articles);
+        setIsLoading(false);
+      })
+      .catch((err) => {
+        setIsError(true);
+      });
+  }, []);
+
+  if (isLoading) {
+    return <p>loading...</p>;
+  }
+
+  if (isError) {
+    return <p>Something went wrong...</p>;
+  }
+
+  return (
+    <main className="content">
+      <h2>content</h2>
```

Comment:
This header's a lil confusing when there's no content on the "/" path 🤔

Reply:
There will be, I've allowed for the home page to be top 5 most recent articles, so home "/" will always have most recent articles on.